site/content/docs/5.3/getting-started/webpack.md

```diff
@@ -130,7 +130,7 @@ With dependencies installed and our project folder ready for us to start coding,
      "scripts": {
        "start": "webpack serve",
        "build": "webpack build",
-       "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Please revert this change or modify the three guides the same way by also adding information about it in the paragraph. Because otherwise, something is missing when you follow the tutorial.

Reply:
Nothing is missing. There's a build script which is just called if one runs test.

Reply:
But anyway, let's keep the moot default for now

Reply:
> Nothing is missing. There's a build script which is just called if one runs test.

Again, the problem was not the modification itself. But in the previous paragraph, something's missing because you didn't explain: "we're gonna change the `test` command to use a basic one to at least build the project blah blah blah". And if we do it, we must do it in the 3 guides.
Let's modify it later on then.